font/src/ft/fc.rs

```diff
@@ -11,42 +11,34 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
-use std::fmt;
-use std::ptr;
-
+use std::{fmt, ptr};
+
+use fontconfig::fontconfig::{
+    FcResultNoMatch, {FcFontList, FcFontMatch, FcFontSort},
+    {FcMatchFont, FcMatchPattern, FcMatchScan}, {FcSetApplication, FcSetSystem},
+    {FC_SLANT_ITALIC, FC_SLANT_OBLIQUE, FC_SLANT_ROMAN},
+    {FC_WEIGHT_BLACK, FC_WEIGHT_BOLD, FC_WEIGHT_EXTRABLACK, FC_WEIGHT_EXTRABOLD},
+    {FC_WEIGHT_BOOK, FC_WEIGHT_MEDIUM, FC_WEIGHT_REGULAR, FC_WEIGHT_SEMIBOLD},
+    {FC_WEIGHT_EXTRALIGHT, FC_WEIGHT_LIGHT, FC_WEIGHT_THIN},
+};
```

Comment:
Can't we remove all brackets in here?

Reply:
I'd be happy to, I just thought it safer to preserve the grouping - as it seemed to me that someone wanted them grouped logically instead of alphabetically.

Reply:
The current grouping does nothing, so there's no point in keeping it.